explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -325,6 +345,8 @@ function InfoSection({ account }: { account: Account }) {
         stakeAccountType={data.parsed.type}
       />
     );
+  } else if (account.details?.owner.toBase58() === NFTOKEN_ADDRESS) {
+    return <NFTokenAccountSection account={account} />;
```

Comment:
This could add up to 100KB gzipped to the explorer, since this is a new hard dependency of the account details page.

https://bundlephobia.com/package/@glow-xyz/nftoken-js@0.1.6

Generally, the architecture of ‘require all the capabilities up front, branch on if/else to select a small subset of those capabilities’ will result in unbounded growth of the Explorer. We'll end up with a distribution of often used and rarely used capabilities, despite paying the cost of downloading the sum total of all capabilities' code on every page load.

Reply:
@steveluscher appreciate that feedback. Do you have a recommendation of how we can avoid that? 

I know Next.js has a `dynamic` import which allows bundle splitting but I'm not sure if you have set something up like that in Explorer.

Reply:
@steveluscher I updated this not to pull in `@glow-xyz/nftoken` by re-implementing all of the parsing and fetching in this library. Lmk if it's good to merge now 👍 :)